IDE/Renesas/e2studio/RX72N/include/user_settings.h

```diff
@@ -49,6 +49,9 @@
     /* #define WOLFBOOT_SIGN_ECC256  */
     /* #define WOLFBOOT_SIGN_ECC384  */
     /* #define WOLFBOOT_SIGN_ECC521  */
+    #if !defined(min)
+    	#define min(data1, data2)    _builtin_min(data1, data2)
+	#endif
```

Comment:
align "endif" to its "if", replace tab with spaces.

Reply:
looks like there's a tab in L53 as well, wguke we use spaces everywhere.

Reply:
Thanks. e2Studio inserts tab as default.